vtex/lean-shipping-calculator
# Use babel to bundle all entrypoints in NPM package

#### What is the purpose of this pull request?

Uses babel to bundle all entrypoints in NPM package. This is the same setup used in `shipping-manager`.

#### What problem is this solving?

Only the `index.js` entrypoint was being bundled before, and thus we couldn't access the exported functions in `modules.js`

#### How should this be manually tested?

You can run `yarn --cwd react build` and check that it does indeed include `index.js` and the other entrypoints in the out directory.

#### Screenshots or example usage

#### Types of changes
